src/uwtools/utils/tasks.py

```diff
@@ -35,7 +35,7 @@ def existing(path: Path):
 @external
 def file(path: Path):
     """
-    An existing file.
+    An existing file or symlink to an existing file.
```

Comment:
`Path.is_file()` returns `True` for files that exist, `True` for symlinks that point to files that exist, and `False` for symlinks that point to files that do not exist.